edx_django_utils/monitoring/internal/utils.py

```diff
@@ -19,9 +19,23 @@
 """
 from contextlib import ExitStack, contextmanager
 
+from django.conf import settings
+
 from .backends import configured_backends
 from .middleware import CachedCustomMonitoringMiddleware
 
+DJANGO_MANAGEMENT_MONITORING_OPERATION_NAME = getattr(
+    settings,
+    'DJANGO_MANAGEMENT_MONITORING_OPERATION_NAME',
```

Comment:
What do you think of the following?
```suggestion
    'DJANGO_MANAGEMENT_MONITORING_FUNCTION_TRACE_NAME',
```